GREEDY/ReceiptCreatings/AutoCorrect.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using GREEDY.Models;
+using NHunspell;
+using System.IO;
+using System.Linq;
+
+namespace GREEDY.ReceiptCreatings
+{
+    class AutoCorrect
```

Comment:
Again classes like these should be public not internal. https://stackoverflow.com/questions/4182015/internal-vs-public-in-c-sharp